viephoe.js

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "VNPhoE",
+        "email": "cv.hoangdong@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://vnphoe.github.io/"
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.